hoot-core-test/hoot-core-test.pro

```diff
@@ -15,6 +15,7 @@ DEPENDPATH += \
     ../tbs/src/main/cpp/ \
     ../tgs/src/main/cpp/ \
     ../hoot-core/src/main/cpp/ \
+    src/test/cpp/ \
```

Comment:
Test project didn't have its own path as a dependency so any use of any header in the project had to be included with a relative path:
`#include "../../TestUtils.h"`
instead of
`#include <hoot/core/TestUtils.h>`